sources/packages/backend/apps/db-migrations/src/sql/EducationProgramsOfferings/Add-col-show-year-of-study.sql

```diff
@@ -0,0 +1,7 @@
+-- Add show_year_of_study to education_programs_offerings table.
+ALTER TABLE
+    sims.education_programs_offerings
+ADD
+    COLUMN offering_note;
```

Comment:
The type is also missing `how_year_of_study BOOLEAN`.
Please ensure that the rollback scripts are tested locally also 😉